spec/cassandra/protocol/cql_byte_buffer_spec.rb

```diff
@@ -71,6 +70,50 @@ module Protocol
           buffer = described_class.new("\xC9v\x8D:")
           expect { buffer.read_varint(7) }.to raise_error(Errors::DecodingError)
         end
+
+        it 'can correctly execute the reference decodings' do
+          described_class.new("\x00").read_varint().should == 0
+          described_class.new("\x01").read_varint().should == 1
+          described_class.new("\x7f").read_varint().should == 127
+          described_class.new("\x00\x80").read_varint().should == 128
+          described_class.new("\x00\x81").read_varint().should == 129
+          described_class.new("\xff").read_varint().should == -1
+          described_class.new("\x80").read_varint().should == -128
+          described_class.new("\xff\x7f").read_varint().should == -129
+        end
```

Comment:
Originally added based on a confusion between varint/vint, but it seemed worthwhile so I kept it around